### DISCUSSION TITLE
XIAO INTRODUCTION | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1425] [Q&A]
# XIAO INTRODUCTION | Seeed Studio Wiki

SeeedStudio XIAO Series Introduction

https://wiki.seeedstudio.com/SeeedStudio_XIAO_Series_Introduction/

**Comment by DarkBrave** [comment]
bluetooth is misspelled in the comparison chart

**Reply by Love4yzp** [reply]
I correct it now.

**Comment by VayuDev** [comment]
There are no symbol & footprint files for the ESP32-C6 :(

**Comment by pop-aruaru** [comment]
XIAO-RP2350 in your shop don't have psram. Is it same your shop XIAO-RP2350?

**Reply by nfs0619** [reply]
Yes, the XIAO RP2350 does not have PSRAM. You can check the product details here: [XIAO RP2350 Product Details](https://www.seeedstudio.com/Seeed-XIAO-RP2350-p-5944.html)

**Reply by pop-aruaru** [reply]
Would you rewrite your wiki.

**Reply by nfs0619** [reply]
Yes, we will immediately review our documentation on the XIAO RP2350 and make the necessary corrections regarding this issue. Thank you for bringing it to our attention!

**Comment by tomtom13** [comment]
Can you please add C5 to comparison table.
